### ISSUE TITLE
🐛 Bug(app/page.tsx): On mobile (i.e. via Android's TalkBack), side menu toggle icon is not accessible, fix and verify other routes, too

### ISSUE THREAD

**Issue by KemingHe** [KemingHe/buckeye-gpt#78]
- [ ] Chat page
- [ ] Auth sign in page
- [ ] Auth verify page (sub-issue: add timeout and alt error)
- [ ] Auth sign out page
